utils/telegraf/Makefile

```diff
@@ -0,0 +1,89 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=telegraf
+PKG_VERSION:=1.19.1
+PKG_RELEASE:=1
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/influxdata/telegraf/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=cec43bb0acfff8b4c963ffec6e3eab44ffb52c8f34e6a697207977cfd05882aa
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+PKG_BUILD_PARALLEL:=1
+
+PKG_MAINTAINER:=Jonathan Pagel <jonny_tischbein@systemli.org>
+PKG_LICENSE:=MIT
+
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/golang/golang-package.mk
+
+MAKE_VARS = \
+	GOARCH=$(GO_ARCH) \
+	LDFLAGS="-s -w"
+
+define Package/telegraf/Default
+	SECTION:=utils
+	CATEGORY:=Utilities
+	TITLE:=Telegraf
+	DEPENDS:=$(GO_ARCH_DEPENDS)
+endef
+
+define Package/telegraf-full
+	$(call Package/telegraf/Default)
+	TITLE+= (Full)
+	VARIANT:=full
+endef
+
+define Package/telegraf
+	$(call Package/telegraf/Default)
+	TITLE+= (Small)
+	VARIANT:=small
+endef
+
+define Package/telegraf/description
+	Telegraf is a plugin-driven agent for collecting and sending metrics and events.
+	It supports various inputs (including prometheus endpoints) and is able to send data into InfluxDB.
+	(Small build. Most plugins excluded)
+endef
+
+define Package/telegraf-full/description
+	Telegraf is a plugin-driven agent for collecting and sending metrics and events.
+	It supports various inputs (including prometheus endpoints) and is able to send data into InfluxDB.
+	(Full build. including all plugins)
+endef
+
+define Build/Compile/telegraf
+	$(CP) ./variant-small/* $(PKG_BUILD_DIR)/
```

Comment:
I suggest copying these files during `Build/Prepare` (after calling `Build/Prepare/Default`). (You should be able to check `$(BUILD_VARIANT)` to see which variant is being built.)